docs/build.gradle

```diff
@@ -63,6 +63,8 @@ buildRestTests.docs = fileTree(projectDir) {
   exclude 'README.asciidoc'
 }
 
+listSnippets.docs = buildRestTests.docs
```

Comment:
This is a "bonus" fix that I made while so that `listSnippets` wouldn't fail. I use `listSnippets` to help make the changes above.